appsec/src/extension/ddappsec.c

```diff
@@ -115,6 +115,9 @@ static zend_extension ddappsec_extension_entry = {
 
 ZEND_GET_MODULE(ddappsec)
 
+#define DD_TAG_SERVER_REQUEST_PATH_PARAMS "server.request.path_params"
```

Comment:
nit: it's not a tag, but rather an address.

Reply:
Done